### ISSUE TITLE
How do I ssh from one container to a different container?

### ISSUE THREAD

**Issue by phil-s** [lando/docs#24]
https://docs.lando.dev/config/ssh.html makes it sound as if this would Just Work, with ssh keys being automatically synchronised between containers, but I'm definitely missing something, because:

* My containers don't have `ssh` or `sshd` installed at all.
* If I install them then `ssh-add -l` shows no keys.
* If I `service ssh start` on a container and try to `ssh -v` to it from another container, key authentication fails and it falls back to asking for passwords.

This is supposed to work, yes?

I'm trying to run a shell command on container B as part of a tool that executes on container A, and ssh is the most obvious way to do that, and it sounded like it wasn't necessary to set things up manually.

**Comment by Xilonz** [comment]
Hmm, that was not the goal of the example, but I don't know how to demonstrate it better. The point of the example is that the docker docker container is able to run commands on the other containers.  I chose to use a separate docker docker container because its easier (and faster) to set up instead of installing docker on one or both containers.

I believe you would need to install docker and mount `/var/run/docker.sock`  inside appserver_1, after that you would be able to excecute commands using `docker exec` like in the example above.

There's no easy way for doing this. You could indeed pipe some outputs together but most of the times this isn't sufficient. 

In my use case I'd like to do something like this;
We run deployer(php) on appserver_1,  and at some point we want to build our javascript and css as part of our deployment process. If we were to install docker into appserver_1 and mount the docker.sock, we should be able to run npm on node_1 by running `docker exec mycrosscontainertoolingexample_node_1 bash -c 'npm run build:prod'`.

**Comment by zahardev** [comment]
@phil-s Agree, SSH to another container would be great. Had the same problem - needed to run composer in node container. 
For now, I decided to use a workaround, maybe it will help.

So, my `package.json` file looks this way:
```
"scripts": {
    "predeploy": "rm -rf vendor/ && composer install --prefer-dist --no-dev && grunt uglify && grunt cssmin && wp-scripts build"
  },
```
  And I wanted to run it with command:
`lando npm run predeploy.`
  
  Of course, I got an error when tried to do so:
`sh: 1: composer: not found`
  
 Workaround - split it into 2 different processes and run a custom lando command that handles both parts: 
 
 Added in .lando.yml:
 
```
  tooling:
  predeploy:
    cmd:
      - appserver: rm -rf vendor && composer install --prefer-dist --no-dev
      - node: grunt uglify && grunt cssmin && wp-scripts build
```

Now I can run the command by short command `lando predeploy`.

**Comment by arnonm** [comment]
Another use case for this would be allowing Visual Studio Code to work in the Remote containers via SSH.
